tests/cpp/test_persistent_buffer.cpp

```diff
@@ -1263,6 +1263,10 @@ TEST_F(PersistentBufferTest, SmemPersistentNotSupportedIn3DReduction) {
 }
 
 TEST_F(PersistentBufferTest, SmemPersistent2DReduction) {
+  // Skip hopper and above as they use cluster reduction
```

Comment:
I remember I asked this before as well, but can you please make sure these tests are kept checked as well? Whether using the cluster reduction or shared memory persistence is a heuristic decision that might depend on particular workloads. We may find cases where shared memory persistence is more efficient, so we should make sure the functionality is kept intact.